docs/study-def-expectations.md

```diff
@@ -98,3 +100,25 @@ The following options are currently available for dummy data (note numeric value
 `"distribution"`(numeric variables) currently has two possible options:
 * `normal`
 * `population_ages`: samples from the distribution of ages in the UK taken from [the Office for National Statistics](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationprojections/datasets/tablea21principalprojectionukpopulationinagegroups).
+
+## Providing your own dummy data
+
+If the expectations framework does not offer enough control over the dummy data that is generated, you can provide your own.
+
+In your `project.yaml`, you can add a `dummy_data_file` value to a `cohortextractor` action.
+
+For instance:
+
+```
+generate_cohort:
+  run: cohortextractor:latest generate_cohort
+  dummy_data_file: test-data/dummy-data.csv
+  outputs:
+    highly_sensitive:
+      cohort: output/input.csv
+
+```
+
+The dummy data file must be commmitted to the repo.
+You should generate the dummy data using a script, and commit the script to the repo.
+You must not use real clinical data as dummy data!
```

Comment:
Maybe worth adding `**`s around this last point, or put in a warning box via `!!! warning`? Not obligatory, but it seems worth emphasising.